index.js

```diff
@@ -1,45 +1,57 @@
 const identity = x => x;
 const getUndefined = () => {};
 const filter = () => true;
-function createRavenMiddleware(Raven, options = {}) {
-  // TODO: Validate options.
-  const {
-    breadcrumbDataFromAction = getUndefined,
-    actionTransformer = identity,
-    stateTransformer = identity,
-    breadcrumbCategory = "redux-action",
-    filterBreadcrumbActions = filter,
-    getUserContext
-  } = options;
+const createRavenMiddleware = (function() {
+  let called = false;
```

Comment:
I think we can just move this to the module scope and avoid the IFFE. The module syntax should handle keeping module variables out of the global scope.